scripts/fetchdep.sh

```diff
@@ -17,7 +17,7 @@ clone() {
     if [ -n "$branch" ]
     then
         echo "Trying to use $org/$repo#$branch"
-        git clone git://github.com/$org/$repo.git $repo --branch "$branch" && exit 0
+        git clone git://github.com/$org/$repo.git $repo --branch "$branch" --depth=1 && exit 0
```

Comment:
Ultra nit, just to parallel how the other option is specified... 😅 

```suggestion
        git clone git://github.com/$org/$repo.git $repo --branch "$branch" --depth 1 && exit 0
```